docs/data-types/lists.md

```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
 ## Examples
```

Comment:
Just a general comment, most of the CLI examples do not use leading whitespace in front of the commands. It's not a big deal, but from a stylistic standpoint, all of examples should be done the same way. Here, there's a mixture.